setup.py

```diff
@@ -22,7 +22,8 @@
         exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
     package_data={'juju': ['py.typed']},
     install_requires=[
-        'macaroonbakery>=1.1,<2.0',
+        # see https://github.com/juju/python-libjuju/issues/914 for macaroon dependency
+        'macaroonbakery@git+https://github.com/cderici/py-macaroon-bakery@relax-protobuf-version#egg=macaroonbakery',
```

Comment:
@SimonRichardson I'd steer away from that because in my mind this is mostly a temporary solution, mostly because 1) I don't want us to maintain that, 2) macaroonbakery library hasn't been touched since 2020, so we gotta find another more up-to-date library for that, wdyt?